File: Asklepios/Asklepios.Core/Entities/Patients/PatientHistory.cs
namespace Asklepios.Core.Entities.Patients;

public class PatientHistory
{
    public Guid HistoryId { get; set; }
    public string PatientName { get; set; }
    public string PatientSurname { get; set; }
    public string PeselNumber { get; set; }
    public string History { get; set; }
}

File: Asklepios.Web/src/models/Patients/patientHistory.ts
export interface PatientVisitDto {
  admissionDate: Date;
  dischargeDate: Date;
  operationName: string;
  result: string;
  complications: string;
  anesthesiaType: string;
}

export interface PatientHistoryDto {
  historyId: string;
  patientName: string;
  patientSurname: string;
  peselNumber: string;
  history: PatientVisitDto[];
}
